lib/navigation.py

```diff
@@ -13,14 +13,14 @@
     close_busy_dialog
 from lib.kodi_formats import is_music, is_picture, is_video, is_text
 from lib.player import TorrestPlayer
-from lib.settings import get_service_ip, get_port, get_buffering_timeout, show_status_overlay, get_min_candidate_size, \
-    ask_to_delete_torrent, download_after_insert, get_files_order, get_metadata_timeout
+from lib.settings import get_service_address, get_port, get_buffering_timeout, show_status_overlay, get_min_candidate_size, \
```

Comment:
The linter is now complaining about this line (125 > 120 characters):
https://github.com/i96751414/plugin.video.torrest/actions/runs/5885372166/job/15961781793?pr=28

Can you fix please?

Reply:
Done